controller/search_blackbox_test.go

```diff
@@ -403,6 +401,140 @@ func (s *searchControllerTestSuite) TestSearchFilter() {
 	assert.Equal(s.T(), "specialwordforsearch", r.Attributes[workitem.SystemTitle])
 }
 
+func (s *searchControllerTestSuite) TestSearchByWorkItemTypeGroup() {
+	s.T().Run(http.StatusText(http.StatusOK), func(t *testing.T) {
+		// given
+		fxt := tf.NewTestFixture(t, s.DB, tf.CreateWorkItemEnvironment())
+		svc := testsupport.ServiceAsUser("TestUpdateWI-Service", *fxt.Identities[0])
+		workitemsCtrl := NewWorkitemsController(svc, gormapplication.NewGormDB(s.DB), s.Configuration)
+		// given work items of different types and in different states
+		type testWI struct {
+			Title          string
+			WorkItemTypeID uuid.UUID
+			State          string
+			SpaceID        uuid.UUID
+		}
+		testWIs := []testWI{
```

Comment:
How about using test fixture for creating all these work items?

Reply:
This will come once we store WIT groups persistently.